<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="FB-133" name="fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest" tests="4" failures="2" timestamp="11 Aug 2020 20:18:13 GMT" time="19.837" errors="0">
  <testcase name="SearchCspPushAlertCon" time="3.575" classname="fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Test Case ID = TC_02Error Massages In Search CSP Push Alert Configuration Failed  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Test Case ID = TC_02Error Massages In Search CSP Push Alert Configuration Failed  expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest.SearchCspPushAlertCon(SearchCspPushAlertConfigurationTest.java:79)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SearchCspPushAlertCon -->
  <testcase name="SearchCspPushAlertCon" time="5.607" classname="fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest"/>
  <testcase name="SearchCspPushAlertCon" time="4.626" classname="fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest"/>
  <testcase name="SearchCspPushAlertCon" time="6.029" classname="fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Test Case ID = TC_01Error Massages In Search CSP Push Alert Configuration Failed  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Test Case ID = TC_01Error Massages In Search CSP Push Alert Configuration Failed  expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest.SearchCspPushAlertCon(SearchCspPushAlertConfigurationTest.java:79)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SearchCspPushAlertCon -->
</testsuite> <!-- fawry.sofAutomation.testsActions.BasicDefinitions.SearchCspPushAlertConfigurationTest -->
